packages/select/src/Select/components/MenuPopup/MenuPopup.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable jsx-a11y/no-autofocus */
+import React from "react"
+import { useOverlay, DismissButton } from "@react-aria/overlays"
+import { FocusScope } from "@react-aria/focus"
+import { mergeProps } from "@react-aria/utils"
+import { useMenuTriggerContext } from "../../provider"
+import styles from "./MenuPopup.module.scss"
+export interface MenuPopupProps {
+  children: React.ReactNode
+}
+
+export const MenuPopup: React.VFC<MenuPopupProps> = ({ children }) => {
```

Comment:
So we can easily find them for a holistic approach later it would be good to rename this to `FloatingSelectWrapper` to match @HeartSquared's naming for Calendar: https://github.com/cultureamp/kaizen-design-system/tree/master/packages/date-picker/src/_subcomponents/FloatingCalendarWrapper